src/Process/Runner/InitialTestsRunner.php

```diff
@@ -47,9 +47,14 @@
  */
 final class InitialTestsRunner
 {
+    private const ERROR_TIMEOUT = 10;
+
     private $processBuilder;
     private $eventDispatcher;
 
+    /**
+     * InitialTestsRunner constructor.
+     */
     public function __construct(
         InitialTestsRunProcessFactory $processFactory,
```

Comment:
hm looks like the property `processBuilder` has not been renamed to `processFactory`, I'll check that out in a separate PR